### PR TITLE
[DowngradePhp80] Add return property downgrade case

### PR DESCRIPTION
**Pull request by TomasVotruba**
Fixing for https://github.com/symplify/monorepo-builder/blob/5e1332f534391014483f0a9ef592e5106d08dbc1/vendor/symfony/console/Style/SymfonyStyle.php#L384

### CODE REVIEW

File: rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+final class ThrowPropertyFetch
+{
+    public function __construct(
+        private ProgressBar|null $progressBar
+    ) {
+    }
+
+    public function run()
+    {
+        if ($this->progressBar === null) {
```
Comment (samsonasik) [inline rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc:37]: it should change to isset instead for `??`

Comment (TomasVotruba) [reply rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc:37]: Indeed. Could you update it?

Comment (samsonasik) [reply rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc:37]: I will try.

Comment (samsonasik) [reply rules-tests/DowngradePhp80/Rector/Expression/DowngradeThrowExprRector/Fixture/throw_property_fetch.php.inc:37]: Fixed at https://github.com/rectorphp/rector-src/pull/1426

**Pull request merged**